mock/test_board.rb

```diff
@@ -0,0 +1,16 @@
+class TestBoard
+  attr_accessor :board, :board_printed
+
+  def initialize(output: output)
+    @board = %w[1 2 3 4 5 6 7 8 9]
+    @board_printed = false
+  end
+
+  def print_board
+    @board_printed = true
+  end
+
+  def mark_board(choice, marker)
+    @board[choice.to_i - 1] = marker
```

Comment:
Should the `TestBoard` know the `choice` parameter needs to be changed to an integer? Noticed the implementation for the `Board` class is the same as well.

Reply:
It knows too much about the other object is what you're saying? I could make the get player input method return an 'int' instead.